forc/src/ops/forc_run.rs

```diff
@@ -56,39 +55,34 @@ pub async fn run(command: RunCommand) -> Result<()> {
 
     if command.dry_run {
         println!("{:?}", tx);
-        Ok(())
+        Ok(vec![])
     } else {
         let node_url = match &manifest.network {
             Some(network) => &network.url,
             _ => &command.node_url,
         };
-        let child = try_send_tx(node_url, &tx, command.pretty_print).await?;
-        if command.kill_node {
-            if let Some(mut child) = child {
-                child.kill().await.expect("Node should be killed");
-            }
-        }
-        Ok(())
+        Ok(try_send_tx(node_url, &tx, command.pretty_print).await?)
     }
 }
 
 async fn try_send_tx(
     node_url: &str,
     tx: &Transaction,
     pretty_print: bool,
-) -> Result<Option<Child>> {
+) -> Result<Vec<fuel_tx::Receipt>> {
     let client = FuelClient::new(node_url)?;
 
     match client.health().await {
-        Ok(_) => {
-            send_tx(&client, tx, pretty_print).await?;
-            Ok(None)
-        }
+        Ok(_) => Ok(send_tx(&client, tx, pretty_print).await?),
```

Comment:
```suggestion
        Ok(_) => send_tx(&client, tx, pretty_print).await,
```

Reply:
I'm only spotting these because clippy complained about a similar thing on master yesterday :smiling_face_with_tear: I wonder why it's not catching these, or if I'm missing something :thinking:

Reply:
:thinking: